pages/reader/pdf.vue

```diff
@@ -8,17 +8,26 @@
     />
     <ClientOnly v-else-if="fileBuffer">
       <PDFReader
+        ref="pdfReaderRef"
         class="grow w-full"
         :book-name="bookInfo.name.value"
+        :nft-class-id="nftClassId"
         :pdf-buffer="fileBuffer"
+        :is-audio-hidden="bookInfo.isAudioHidden.value"
         :book-file-cache-key="bookFileCacheKey"
         @error="handlePDFError"
+        @pdf-loaded="handlePDFLoaded"
+        @tts-play="onClickTTSPlay"
```

Comment:
Please keep naming consistency for event handler function
```suggestion
        @tts-play="handleTTSPlay"
```